apps/status-api/src/controllers/OracleStatusController.ts

```diff
@@ -27,6 +27,29 @@ export class OracleStatusController {
     }, 5) // cache status result for 5 seconds
   }
 
+  /**
+   * To provide the status of each ticker determined by the number of oracles responded given the ticker id
+   *
+   * @return {Promise<OracleStatus>}
+   * @param token
+   * @param currency
+   */
+  @Get('/:token/:currency')
+  async getOracleRespondStatus (@Param('token') token: string, @Param('currency') currency: string): Promise<{ status: OracleStatus }> {
+    return await this.cachedGet(`oracle-${token}-${currency}`, async () => {
+      const oracles = await this.client.prices.getOracles(token, currency, 60)
+      const total = oracles.filter(oracle => oracle.feed !== undefined).length
+
+      const prices = await this.client.prices.get(token, currency)
+      const active = prices.price.aggregated.oracles.active
+
+      if (active > 3 || (active / total) >= 0.75) {
```

Comment:
Do we want it as operational for which of the following condition?
1. there are 3 active OR at least 75% active
2. there are 3 active AND at least 75% active

Reply:
There are 3 active AND at least 75% active

Reply:
The 75% condition will not be a good gauge for tickers that recently went through stock-split as the total oracle number will include the pre-split ones. Thinking of an alternative condition now

Reply:
Conclusion: 
If there are >= 3 total oracles, alert if the active oracles >=3.
If there are 3 total oracles (outlier case), alert if the active oracles >3.

We will remove the 75% condition